internal/services/firewall/firewall_resource.go

```diff
@@ -164,6 +164,12 @@ func resourceFirewall() *pluginsdk.Resource {
 					}
 					return string(network.AzureFirewallThreatIntelModeAlert)
 				}(),
+				Computed: func() bool {
```

Comment:
Computed has been added because the default was removed with 3.0 but azure defaults this to different values depending on how the firewall was created. 

The question is whether this should instead be Required

Reply:
what's the difference between `Off` and `""`? should we be removing the value `Off` in favour of omitting the field?

Reply:
Keeping it in after discussing internally